File: tests/fixtures/patterns-positive/facade/mutB.py
# POSITIVO (mutación B: sin Facade; secuencia repetida en 2 llamadores).
def checkout(order, payment, inventory, shipping, notifier):
    payment.charge(order.total)
    inventory.reserve(order.items)
    shipping.schedule(order.date)
    notifier.send_confirmation()


def checkout_express(order, payment, inventory, shipping, notifier):
    payment.charge(order.total)
    inventory.reserve(order.items)
    shipping.schedule(order.date)
    notifier.send_confirmation()

File: tests/fixtures/patterns-positive/facade/mutB.ts
// POSITIVO (mutación B: sin Facade; secuencia repetida en 2 llamadores).
function checkout(order: Order, payment: PaymentGateway, inventory: InventoryService, shipping: ShippingProvider, notifier: NotificationCenter): void {
  payment.charge(order.total);
  inventory.reserve(order.items);
  shipping.schedule(order.date);
  notifier.sendConfirmation();
}
function checkoutExpress(order: Order, payment: PaymentGateway, inventory: InventoryService, shipping: ShippingProvider, notifier: NotificationCenter): void {
  payment.charge(order.total);
  inventory.reserve(order.items);
  shipping.schedule(order.date);
  notifier.sendConfirmation();
}
